{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/email.py

```diff
@@ -26,24 +26,28 @@ def create_attachment(path: str, mime_type: MIMEType = MIMEImage) -> MIMEType:
 
 
 def send_mail(
-    template_name: str,
     subject: str,
-    body: str,
     to: list[str],
     from_: str = f'<{settings.DEFAULT_FROM_EMAIL}>',
+    cc: Optional[list[str]] = None,
+    template_name: Optional[str] = None,
+    body: Optional[str] = None,
     context: Optional[dict] = None,
     attachments: Optional[list[str]] = None,
 ):
+    # Ensure that either body or template name is provided, not both.
+    assert (body is not None and template_name is None) or (body is None and template_name is not None)
```

Comment:
It's a shame Python doesn't have an exclusive or. :D

Reply:
I could go with something like
```python
has_body = body is not None
has_template = template_name is not None
assert has_body != has_template
```
Do you find it more suiting?

Reply:
Removed, as it wasn't relevant any more.